cEP-0032.md

```diff
@@ -0,0 +1,651 @@
+# Enhance Websites Architecture
+
+| Metadata |                                                       |
+| -------- | ------------------------------------------------------|
+| cEP      | 0032                                                  |
+| Version  | 1.0                                                   |
+| Title    | Enhance Websites Architecture                         |
+| Authors  | Keshav Garg <mailto:co16326.ccet@gmail.com>           |
+| Status   | Work In Progress                                      |
+| Type     | Feature                                               |
+
+## Abstract
+
+This document describes the features that will increase the usability and
+easiness for all the open-source enthusiasts by enhancing and improving the
+coala Community website as well as coala Projects website for newcomers,
+developers as well as for other communities. In addition, new API Endpoints
+will be created in coala webservices. These new endpoints will manage the user
+-database as well as coala database in a more structured form.
+
+## Introduction
+
+A website plays an essential role for an open-source community because it
+spreads the idea of offerings that community provide to other organizations,
+with information about the active community by exploring their skills as well
+as their precious contributions to the organization. And to provide such
+offerings this project aims to provide useful features to community members as
+well as to other developers and organizations. They can use the below
+mentioned features to be a part of coala community and can even get information
+about coala community and its contributors who have been part of SoC programs.
+
+## coala [Community website](community.coala.io/)
+
+### Enhancements
+
+The listed enhancements will ease the navigatibility and usability for
+newcomers, developers and to all others who wanted to be a part of coala
+organization. It will help community members to know about their performance
+in-comparison to other members. Following are the new enhancments that will be
+added to coala community website:
+
+1. **Display a webpage for-**
+   1. **organization teams**
+
+        This will show information about all [coala teams](https://github.com/orgs/coala/teams).
```

Comment:
Sounds interesting, but that link is private and only available to the community members. Not sure if it is a good idea to make it public.

Reply:
Took idea from https://github.com/coala/community/issues/12

Reply:
the web-page will be showed to just logged-in users, now